src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -62,8 +63,14 @@ interface WorkspaceFormProps {
   application: ApplicationStart;
   onSubmit?: (formData: WorkspaceFormData) => void;
   defaultValues?: WorkspaceFormData;
+  opType?: string;
```

Comment:
How about update `opType?: string` to `opType?: WORKSPACE_OP_TYPE_CREATE | WORKSPACE_OP_TYPE_UPDATE `